.eslintignore

```diff
@@ -1,2 +1,3 @@
 vendor
 node_modules
+src/Views/Form/Themes/Sequoia/assets
```

Comment:
@henryholtgeerts is there any specific reason to exclude from them from eslint?

Reply:
Nope, forgot to remove this line from this file - was for testing purposes. I have now removed it, ref: https://github.com/impress-org/givewp/commit/a75c76af2e62daa39d009d3ee451b10ab64ada7d